resolvers/updateItem.js

```diff
@@ -252,6 +253,12 @@ const updateItem = async (
 			timeItTook,
 			position,
 			dueDate,
+			tags:
+				tags && tags.length > 0
```

Comment:
```suggestion
				tags
```

We should keep the possibility to remove all tags from the item.